src/azure-cli/azure/cli/command_modules/apim/custom.py

```diff
@@ -151,18 +155,28 @@ def get_apim_api(client, resource_group_name, service_name, api_id):
     return client.api.get(resource_group_name, service_name, api_id)
 
 
-def list_apim_api(client, resource_group_name, service_name):
+def list_apim_api(client, resource_group_name, service_name, filter_displayName=None, top=None, skip=None, include_not_tagged_apis=None):
     """List all APIs of an API Management instance. """
 
-    return client.api.list_by_service(resource_group_name, service_name)
+    if filter_displayName is not None:
+        filter_displayName = "properties/displayName eq '%s'" % filter_displayName
+
+    return client.api.list_by_service(resource_group_name, service_name, filter=filter_displayName, skip=skip, top=top, include_not_tagged_apis=include_not_tagged_apis)
 
 
-def delete_apim_api(client, resource_group_name, service_name, api_id, delete_revisions=True, no_wait=False):
+def delete_apim_api(client, resource_group_name, service_name, api_id, delete_revisions=None, if_match=None, no_wait=False):
     """Deletes an existing API. """
 
     cms = client.api
 
-    return sdk_no_wait(no_wait, cms.delete, resource_group_name=resource_group_name, service_name=service_name, api_id=api_id, if_match='*', delete_revisions=delete_revisions)
+    return sdk_no_wait(
+        no_wait,
+        cms.delete,
+        resource_group_name=resource_group_name,
+        service_name=service_name,
+        api_id=api_id,
+        if_match="*" if if_match is None else if_match,
+        delete_revisions=delete_revisions if delete_revisions is not None else False)
```

Comment:
Does it a breaking change? The default value is `True` before.

Reply:
nope, it is not a breaking change, it should be false as default value, the previous contributor made a mistake here,